src/cron/exportToTinybird.ts

```diff
@@ -22,16 +22,17 @@ const cron: Cron = {
     logger.info(`fetching post changes since ${latest.toISOString()}`);
     const posts = await con.query(
       `SELECT "id",
-              "authorId" AS "author_id",
-              "createdAt" AS "created_at",
-              "metadataChangedAt" AS "metadata_changed_at",
-              "creatorTwitter" AS "creator_twitter",
-              "sourceId" AS "source_id",
-              "tagsStr" AS "tags_str",
-              ("banned" or "deleted") ::int AS "banned"
+              "authorId"                    AS "author_id",
+              "createdAt"                   AS "created_at",
+              "metadataChangedAt"           AS "metadata_changed_at",
+              "creatorTwitter"              AS "creator_twitter",
+              "sourceId"                    AS "source_id",
+              "tagsStr"                     AS "tags_str",
+              ("banned" or "deleted") ::int AS "banned",
+              "type"                        AS "post_type",
+              "private"::int                AS "source_private"
```

Comment:
@vpol I think we changed the name of this column right?

Reply:
yes, fixed. forgot about it.